### PR TITLE
:zap: Major re-work of command handler (Automatic)

### PR DESCRIPTION
**Pull request by Townsy45**
* :bug: Fixed bug with guildPrefix so now it should not error if the guildPrefix / guild does not exist in the database.

### CODE REVIEW

File: src/cmd/index.ts [hunk 1]
```diff
@@ -1,27 +1,81 @@
...
     }
 }
 
 export default Commands
+
+async function fetchCommands() {
+    let cmdFiles = await getCommands();
+    let cmds = [];
+    for (const cmd of cmdFiles)
+        cmds.push(require(cmd.filePath).default);
```
Comment (jw2476) [inline src/cmd/index.ts:33]: You could instantiate the class here

**Review by jw2476 — CHANGES_REQUESTED** [review]
The functions and variables need type hinting

**Comment by Townsy45** [comment]
I have moved where the classes are initialised but I disagree with the typing.

When I try to assign types to this is produces a lot of errors for stuff that actually works. It would be more work for no benefit.

Feel free to take a look yourself but you will have to rewrite most of it to fit specific typing.

**Comment by jw2476** [comment]
Not all of it needs to be typeed, only the functions returns and variables at the top of the file.

**Comment by jw2476** [comment]
@Townsy45 would you like me to make the changes? The command handler is really important and I'm more then happy to do it if you don't want to!

**Comment by Townsy45** [comment]
Changing PR from Master => Beta

**Pull request closed without merging**